openedx/core/djangoapps/schedules/signals.py

```diff
@@ -93,6 +114,16 @@ def update_schedules_on_course_start_changed(sender, updated_course_overview, pr
         )
 
 
+def _suppress_schedules_for_some_enrollments(schedule_config):
+    if schedule_config.hold_back_ratio <= 0 or schedule_config.hold_back_ratio > 1:
```

Comment:
Can we use django admin form validation instead (to validate at time of saving in the admin)? https://docs.djangoproject.com/en/1.11/ref/contrib/admin/#admin-custom-validation

Otherwise, we'll assume suppression is working when the admin config is saved.